students/ZidanLuo/session02/series.py

```diff
@@ -0,0 +1,40 @@
+import sys
```

Comment:
careful with your indentation!!

you want four spaces per indent. always.

Make sure your editor is set up to do that when you hit the <tab> key...